fable_agents/server.py

```diff
@@ -65,6 +65,18 @@ async def message(sid, message_type, message_data):
         parsed_data = message_data
     msg = models.Message(message_type, parsed_data)
 
+    if msg.type == 'new-memories':
+        memories = msg.data.get('memories', [])
+
+        reset = msg.data.get('reset', True)
+        if reset:
+            Datastore.memories.reset()
+
+        for memory in memories:
+            memory = models.Memory.from_dict(memory)
+            API.agents.get(memory.context_id).memories().append(memory)
+            print("MEMORY ADDED:", memory)
+
     if msg.type == 'choose-sequence':
```

Comment:
I think this needs to be an 'elif' - it's creating this warning right now: 
handler not found for message type:new-memories {'memories': [{'timestamp': '1999-12-23T00:00:00', 'summary': "I've been working with the Mayor to stop crime in Thistle Gulch.", 'contextId': 'wyatt_cooper', 'entityIds': ['wyatt_cooper', 'jack_kane'], 'position': '{"x":0.0,"y":0.0,"z":0.0}'}], 'reset': True}